occurrences/admin.py

```diff
@@ -193,7 +196,7 @@ def linked_event_id(self, obj):
 
 
 @admin.register(EventQueueEnrolment)
-class EventQueueEnrolmentAdmin(admin.ModelAdmin):
+class EventQueueEnrolmentAdmin(AuditlogAdminViewAccessLogMixin, admin.ModelAdmin):
```

Comment:
As EventQueueEnrolmentAdmin's list_display contains get_person which shows the person's name (See screenshot for example) which is GDPR data, wouldn't it make sense to log the list views because of this?
```suggestion
class EventQueueEnrolmentAdmin(AuditlogAdminViewAccessLogMixin, admin.ModelAdmin):
    enable_list_view_audit_logging = True
```
![image](https://github.com/user-attachments/assets/3023c08d-35cf-4049-9c04-3e270184f91a)

Reply:
I understand your logic, but I still would not enable the `enable_list_view_audit_logging` since the person name is the only information there. It's shared information anyway and it would fool the auditlog readers, that somebody really saw all the contact information, etc. Writing access log of these would just spam the auditlog, wouldn't it?

Reply:
How about masking the person column?